content/academy/data_collection_basics/node_continued.md

```diff
@@ -8,115 +8,118 @@ paths:
+In the first part of the Node.js tutorial we downloaded the HTML of our <a href="https://commerce-qd83plqbj-mstephen19.vercel.app/search/on-sale" target="_blank">Morgan Webstore e-commerce site</a> and parsed it with Cheerio. Now, we will replicate the collection logic from the [Collecting Data with DevTools]({{@link data_collection_basics/using_devtools.md}}) chapters and finish our scraper.
 
 ## [](#querying-with-cheerio) Querying data with Cheerio
 
-As a reminder, the data we need from the Top Sites index is available in the 50 `<div>` elements with class `site-listing`. The CSS selector to find those is `div.site-listing`.
+As a reminder, the data we need for each product on the page is available in each `a[href*="/product/"]` element.
 
-![Selecting an element from the Elements tab]({{@asset data_collection_basics/images/selecting-container-element.webp}})
+![Selecting an element from the Elements tab]({{@asset data_collection_basics/images/selecting-container-element-new.png}})
 
 To get all the elements with that CSS selector using Cheerio, we call the `$` function with the selector.
 
-```js
-$('div.site-listing');
+```JavaScript
+$('a[href*="/product/"]');
 ```
 
-We will use the same approach as in the previous DevTools chapters. Using a `for..of` loop we will iterate over the array of sites we saved in the `sites` variable. The code is a little different from DevTools, because we're using Node.js and Cheerio not a browser.
+We will use the same approach as in the previous DevTools chapters. Using a `for..of` loop we will iterate over the array of products we saved in the `products` variable. The code is a little different from DevTools, because we're using Node.js and Cheerio (which used jQuery syntax) instead of a browser (using Vanilla JavaScript syntax).
 
-```js
+```JavaScript
 // main.js
 import { gotScraping } from 'got-scraping';
 import cheerio from 'cheerio';
 
-const response = await gotScraping('https://www.alexa.com/topsites');
+const response = await gotScraping('https://commerce-qd83plqbj-mstephen19.vercel.app/search/on-sale');
 const html = response.body;
 
 const $ = cheerio.load(html);
-const sites = $('div.site-listing');
-for (const site of sites) {
-    const element = $(site);
-    console.log(element.text());
+const products = $('a[href*="/product/"]');
+
+for (const product of products) {
+    const element = $(product);
+
+    console.log(element.textContent);
 }
 ```
 
-After you run this script, you should see data of all the 50 sites printed in your terminal. Don't forget about the `const element = $(site);` line. Without wrapping each `site` with `$()`, we wouldn't be able to call the `.text()` function on it.
+After you run this script, you should see data of all the 32 products printed in your terminal. Don't forget about the `const element = $(product);` line. Without wrapping each `product` with `$()`, we wouldn't be able to call the `.text()` method on it.
```

Comment:
It's debatable whether using`method` is the correct approach here. Some beginners might not know what a method is, but would be perfectly fine with `function`